charts/minio-lifecycle-mgr/values.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+lifecycleMgr:
+  minio:
+    endpoint: http://minio:9000
+    secretName: minio
+  buckets:
+#    - name: cortex
+#      expirationDays: 15
+#    - name: loki
+#      expirationDays: 15
```

Comment:
I guess this file can be removed since we have config/minio-lifecycle-mgr/values.yaml ?

Reply:
oh sorry, this is the values.yaml in the chart. then it LGTM

Reply:
right :)